addons/payment_mollie/const.py

```diff
@@ -12,7 +12,7 @@
 ]
 
 # Currency codes in ISO 4217 format supported by mollie.
-# See https://docs.mollie.com/payments/multicurrency
+# See https://docs.mollie.com/payments/multicurrency. Last seen online: 22 September 2022.
```

Comment:
Maybe it could be useful to add a note to inform that the availability of those currencies varies per payment method.